src/main/java/com/dataguise/saas/exception/InternalServerException.java

```diff
@@ -0,0 +1,41 @@
+package com.dataguise.saas.exception;
+
+import org.springframework.http.HttpStatus;
+
+/**
+ * This exceptions are thrown when there is an error in processing from saas-ui backend to frontend.
+ */
+public class InternalServerException extends Exception {
+
+    String errorMessage;
+    HttpStatus status;
+
+    public InternalServerException() {}
+
+    public InternalServerException(String message, HttpStatus status) {
+        super(message);
+        this.errorMessage = message;
+        this.status = status;
+    }
+
+    public InternalServerException(String message) {
+        super(message);
+        this.errorMessage = message;
+    }
+
+    public HttpStatus getStatus() {
+        return status;
+    }
+
+    public void setStatus(HttpStatus status) {
```

Comment:
**Reporter**: Checkstyle    
    **Rule**: com.puppycrawl.tools.checkstyle.checks.javadoc.MissingJavadocMethodCheck
    **Severity**: ERROR
    **File**: /work/src/main/java/com/dataguise/saas/exception/InternalServerException.java L30
    
    Missing a Javadoc comment.
    
*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1581015510>*